src/Microsoft.VisualStudio.Editors/ResourceEditor/ResourceEditorRefactorNotify.vb

```diff
@@ -11,11 +11,6 @@ Imports Microsoft.VSDesigner.Common
 
 Namespace Microsoft.VisualStudio.Editors.ResourceEditor
 
-    ''' <summary>
-    ''' The editor factory for the resource editor.  The job of this class is
-    '''   simply to create a new resource editor designer when requested by the
-    '''   shell.
-    ''' </summary>
```

Comment:
Is this summary not accurate?

Reply:
Removed in 7bd52f16a267a3a99af0d90830c16996aaf83ece. It was a copy/paste from a related class, and was not accurate.